docs/USAGE.md

```diff
@@ -84,7 +106,7 @@ It works as you would expect, though the type is scary. Think of it as the Haske
 fix :: ((a -> b) -> (a -> b)) -> a -> b
```

Comment:
I _believe_ both of these signatures can be written as

```haskell
pfix :: Term s (((a :--> b) :--> (a :--> b)) :--> (a :--> b))
```

and

```haskell
fix :: ((a -> b) -> (a -> b)) -> (a -> b)
```

which, to me, makes it clear that there isn't some hidden difference between the signatures and highlights that they're working on functions in particular. This is also how `pfix` is listed in GHCi.